package.json

```diff
@@ -59,6 +60,7 @@
   },
   "devDependencies": {
     "@openeth/truffle-typings": "^0.0.6",
+    "@openzeppelin/test-helpers": "0.5.10",
```

Comment:
I think we don't need it here, if we have it in the main dependencies, but I created a [task](https://rsklabs.atlassian.net/browse/PP-442) to handle this